modules/etc/generated/wildfly/modules/system/layers/fuse/com/braintreepayments/gateway/braintree/main/module.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.1" name="com.braintreepayments.gateway.braintree">
+  <properties>
+    <property name="jboss.api" value="private" />
+  </properties>
+  <resources>
+    <resource-root path="braintree-java-2.53.0-javadoc.jar" />
```

Comment:
I think initially we have no choice but to include this as the smartics plugin can only handle matches and exclusions based on groupId and artifactId. There is no option to match on classifier or scope.